src/ORM/Table.php

```diff
@@ -2304,4 +2305,26 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateAtLeast($value, $expectedCount = 0, $operator = '>')
```

Comment:
Maybe others can also join into the discussion. But I think the operator is not needed if we can adjust the count from 1 ... n.